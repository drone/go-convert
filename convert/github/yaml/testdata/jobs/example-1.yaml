# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-creating-jobs

jobs:
  my_first_job:
    name: My first job
  my_second_job:
    name: My second job