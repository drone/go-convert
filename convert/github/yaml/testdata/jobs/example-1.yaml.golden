jobs:
  my_first_job:
    name: My first job
  my_second_job:
    name: My second job